src/components/NotificationCenter/Notification.js

```diff
@@ -21,11 +21,12 @@ export default function Notification({onDismiss, onDeactivate, notification, onS
                   </span>
                 </React.Fragment>)
               }
+              style={{ marginLeft: -11 }}
             >
-              <DropdownItem onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 1)}>Snooze 1 day</DropdownItem>
-              <DropdownItem onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 3)}>Snooze 3 days</DropdownItem>
-              <DropdownItem onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 7)}>Snooze 1 week</DropdownItem>
-              <DropdownItem onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 21)}>Snooze 3 weeks</DropdownItem>
+              <DropdownItem style={{ textAlign: 'left' }} onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 1)}>Snooze 1 day</DropdownItem>
+              <DropdownItem style={{ textAlign: 'left' }} onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 3)}>Snooze 3 days</DropdownItem>
+              <DropdownItem style={{ textAlign: 'left' }} onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 7)}>Snooze 1 week</DropdownItem>
+              <DropdownItem style={{ textAlign: 'left' }} onClick={() => onSnooze(notification.id, notification.todoListId, notification.todo, 21)}>Snooze 3 weeks</DropdownItem>
```

Comment:
Bulma have got you covered 
Try this instead https://bulma.io/documentation/components/dropdown/#right-aligned

Reply:
The class must be set on the dropdown. 
You will have to use an additional property called `align` (you can ofc choose another word if you prefere). You can use [React default props](https://en.reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) to basically set this property to 'left'. 
When 'right' is provided, the component should add an additional class on the dropdown element. 
Take a look (and use ! ) this, it will help you https://github.com/JedWatson/classnames

Reply:
che ne compwend pas :'(